runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -72,6 +76,11 @@ public Dimension render(Graphics2D graphics)
 			return null;
 		}
 
+		if (onInventoryTab && config.hiddenInventory())
```

Comment:
This can be merged with the condition above.

```java
if (hidden || (plugin.isOnInventoryTab() && config.hiddenInventory()))
{
	return null;
}
```